src/models/fhir/constraints.ts

```diff
@@ -131,10 +131,10 @@ const artGroupedFhirKeys = {
 };
 
 /**
- * PCR constraints:
+ * PCR or LAMP constraints:
  * To convert parsed format -> original FHIR Bundle format (to aid in custom error message)
  */
-const pcrGroupedFhirKeys = {
+const pcrOrLampGroupedFhirKeys = {
```

Comment:
Btw this is also for SER constraints right? Shall we name it `pcrSerLampGroupedFhirKeys`?